pkg/instancetype/instancetype_test.go

```diff
@@ -1053,6 +1053,18 @@ var _ = Describe("Instancetype and Preferences", func() {
 				Expect(*vmi.Spec.Domain.Memory.Hugepages).To(Equal(*instancetypeSpec.Memory.Hugepages))
 			})
 
+			It("should apply memory overcommit corretly to VMI", func() {
```

Comment:
typo which will solve code-lint job:
```suggestion
			It("should apply memory overcommit correctly to VMI", func() {
```